introspect-schema

```diff
@@ -0,0 +1,36 @@
+#! /usr/bin/env node
+
+const { introspectionQuery } = require('graphql');
```

Comment:
This executable requests the introspected schema and writes it to `src/build` for use in the populate exchange. It is triggered pre build/start in package.json